src/components/TopBar.js

```diff
@@ -1,12 +1,13 @@
-import React, { Component } from 'react';
+import React from 'react';
 import styled from 'styled-components';
 import Button from './Button';
 import { connect } from 'react-redux';
 import { login, logout } from './../actions';
 import Text from './Text';
 import Colors from '../styles/Colors';
+import PropTypes from 'prop-types';
 
-class TopBar extends Component {
+class TopBar extends React.Component {
```

Comment:
Why?

Reply:
@Tuhaj it does not matter - both effect is the same (i.e. either importing `React, {Component}` or just `React`). The only reason why I picked `React.Component` is that `Component` is a generic name that any library may export. By using `React.Component`  we strictly say "hey, this will be React.Component!" - as `Component` is a common name and may be used by other libraries.